### ISSUE TITLE
ncu --doctor does not consider --packageManager

### ISSUE THREAD

**Issue by Pezmc** [raineorshine/npm-check-updates#730]
- [x] I have searched for [similar issues](https://github.com/raineorshine/npm-check-updates/issues)
- [x] I am using the latest version of `npm-check-updates`
- [x] I am using `node >= 10.17`

---------------------------

## Steps to Reproduce

Run `ncu --doctor --packageManager yarn -u`

## Current Behavior

It runs commands against npm as lib/doctor always spawns an npm process

https://github.com/raineorshine/npm-check-updates/blob/c68768ac77dd4dcf160918f15fb1ae72fd0b8452/lib/doctor.js#L8

## Expected Behavior

It would run commands against yarn instead, i.e.

```
yarn install
yarn run test
ncu -u
yarn install
yarn run test
```

**Comment by raineorshine** [comment]
Thanks for reporting! This should be pretty straightforward. I'm not sure the implications of omitting `--no-save`, which is [not supported by yarn](https://github.com/yarnpkg/yarn/issues/1743). I'll do a direct replacement of `npm` with `yarn` and you can let me know if it works as expected.

**Comment by raineorshine** [comment]
Published in `v8.1.0`

**Comment by Pezmc** [comment]
Amazing, already making use of it! 

Closed by df40ac4b6c250e2a4fb6fe83b62f3e2b9296595c
